config/mock-api-data.js

```diff
@@ -11,6 +11,26 @@ function * range (max)
     return max;
 }
 
+function setTagName () {
+    
+    const tagsArray = [
+        'florist',
```

Comment:
I'd consider leaveraging faker for this -> maybe lorem.word / random.word would be a good choice? ;) [but not insisting]

Reply:
Not outdated